optimum/exporters/onnx/convert.py

```diff
@@ -323,6 +328,34 @@ def export_pytorch(
                 opset_version=opset,
             )
 
+            # check if external data was exported
+            onnx_model = onnx.load(str(output), load_external_data=False)
+            model_uses_external_data = check_model_uses_external_data(onnx_model)
+
+            if model_uses_external_data or FORCE_ONNX_EXTERNAL_DATA:
+                logger.info("Saving external data to one file...")
+
+                # try free model memory
+                del model
+                del onnx_model
+
+                onnx_model = onnx.load(
+                    str(output), load_external_data=True
+                )  # TODO: this will probably be too memory heavy, shall we free `model` memory?
+                onnx.save(
+                    onnx_model,
+                    str(output),
+                    save_as_external_data=True,
+                    all_tensors_to_one_file=True,
+                    location=output.name + "_data",
+                    size_threshold=1024 if not FORCE_ONNX_EXTERNAL_DATA else 0,
```

Comment:
Any use_case why we want the threshold to be 0 or `FORCE_ONNX_EXTERNAL_DATA` in general.